### ISSUE TITLE
[MozFest] Add dividing lines to Image teaser block

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#9535]
Blocked by https://github.com/mozilla/foundation.mozilla.org/issues/9400 & based on design done in https://github.com/mozilla/foundation.mozilla.org/issues/9504

This is part of the work for @marcwalsh for the MozFest site refresh. 

Acceptance Criteria

- [ ] Diving lines should look and function similar to the `Image text` block (see screen cap below)
- [ ] As an editor, I can select a check box "top divider" on the `Image teaser` block to show a line at the top of the block
- [ ] As an editor, I can select a check box "bottom divider" on the `Image teaser` block to show a line at the bottom of the block
- [ ] As an editor, if I check neither check box, no divider lines show up. (Both lines are be optional.)
- [ ] As a user, when I can visit a page with two (more than one) `Image teaser` block(s) on it with divider lines shown, the spacing between the line and the block content should look even on both sides of a line between blocks (equal above and below as in screen cap from Figma)
- [ ] Other elements from https://github.com/mozilla/foundation.mozilla.org/issues/9400 (text, image, responsiveness, button, etc) should continue to function and look the same  

Screen cap of option diving lines in our `Image text` block

<img width="1446" alt="image" src="https://user-images.githubusercontent.com/1068783/196751874-344d12e3-bdcc-4c0b-9806-08d4c8f2742f.png">
 
Screen cap of the `Image teaser` block & [the figma file](https://www.figma.com/file/LV5mF18AkBCMOyk4692S1j/MozFest-updates?node-id=408%3A2848).

<img width="1239" alt="image" src="https://user-images.githubusercontent.com/1068783/197816320-889a1e98-0d97-4333-90f3-bec40a1fd340.png">

**Comment by tbrlpld** [comment]
@kristinashu Can you add some more details as to what is currently missing from the teaser block?

**Comment by kristinashu** [comment]
@tbrlpld everything should have been done in https://github.com/mozilla/foundation.mozilla.org/issues/9400 with the exception of the diving lines. Note, the lines were a new request from a different stakeholder - not the result of missing ACs :)

I've updated one more AC to the list but please let me know what else I should add to give this more clarity.

**Comment by tbrlpld** [comment]
@kristinashu I have trying to reformulate the ACs to how I understood them. Can you check that this matches your understanding?

**Comment by kristinashu** [comment]
Perfect!

**Comment by tomusher** [comment]
Resolved by #9598 (pending review).

The existing `ImageText` block lines are black, so I have copied the style of those for consistency rather than copying the Figma design where the lines are grey.

**Comment by kristinashu** [comment]
Thank you for catching that Tom, black lines sounds good to me.

**Comment by marcwalsh** [comment]
Hey @tomusher @kristinashu just took a look at this in staging. This does seem great for what we need - just one flag is the image doesn't centre vertically when longer text is posted. This is an extreme example and I'll be cutting down the amount of text, but shows the issue:

<img width="1213" alt="Screenshot 2022-11-03 at 15 55 42" src="https://user-images.githubusercontent.com/30226684/199810722-61e52112-b122-4bec-9a08-1ced7da87705.png">

**Comment by cdanfon** [comment]
Closing this issue as complete, but opening a new ticket to address the image positioning as a separate job.
